mne/stats/tests/test_parametric.py

```diff
@@ -30,34 +31,35 @@
 
 
 def generate_data(n_subjects, n_conditions):
-    """generate testing data"""
+    """Generate testing data."""
     rng = np.random.RandomState(42)
     data = rng.randn(n_subjects * n_conditions).reshape(
         n_subjects, n_conditions)
     return data
 
 
 def test_map_effects():
-    """ Test ANOVA effects parsing"""
+    """Test ANOVA effects parsing."""
     selection, names = _map_effects(n_factors=2, effects='A')
-    assert_true(names, ['A'])
+    assert names == ['A']
 
     selection, names = _map_effects(n_factors=2, effects=['A', 'A:B'])
-    assert_true(names, ['A', 'A:B'])
+    assert names == ['A', 'A:B']
 
     selection, names = _map_effects(n_factors=3, effects='A*B')
-    assert_true(names, ['A', 'B', 'A:B'])
+    assert names == ['A', 'B', 'A:B']
 
+    # XXX this might be wrong?
     selection, names = _map_effects(n_factors=3, effects='A*C')
-    assert_true(names, ['A', 'B', 'A:B', 'C', 'A:C', 'B:C', 'A:B:C'])
+    assert names == ['A', 'B', 'A:B', 'C', 'A:C']
```

Comment:
@dengemann this is your test from a long time ago. The old check didn't really check anything meaningful (just that `names` was not empty). This is what I had to change the check to for it to pass. But this does not look correct to me. If so, could you fix it in a follow-up PR?